coloured heading

nettle790 opened the pull request:
added a colour to the heading.

nettle790 committed:
coloured heading

codeTrackLift reviewed (changes requested):
Please follow notes as outlined in Issue #4 

create styles folder
name the stylesheet styles.css and place in the styles folder
add a background
add a header with some intro text

nettle790 committed:
added stylesheet

nettle790 commented:
issue #4 

please review the changes

styles folder created within which styles.css file has been created which is linked to index.html.

applied a background colour and an  image.

provided a header

codeTrackLift commented on index.html:
```diff
@@ -1,12 +1,17 @@
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+  <link rel="stylesheet" href="styles/styles.css" type="text/css">
```
suggest to prefix url w `./`

is the `type="text/css"` necessary?

codeTrackLift replied:
@nettle790

codeTrackLift replied:
Please prefix href with `./` in line 9

codeTrackLift commented on index.html:
```diff
@@ -1,12 +1,17 @@
+
+<body>
+  <h1 style="color:red;">Hello World!</h1>
+  <h2>ATHENA the patron goddess of Greece<h2><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br></br>
```
Any reason for the `<br>`s?

If extra space is needed, suggest to apply styling via added stylesheet.

codeTrackLift replied:
@nettle790

codeTrackLift commented on styles/styles.css:
```diff
@@ -0,0 +1,7 @@
+body {
+  background-color: pink;
+  text-align: center;
+  background-image: url("athens0.jpeg");
```
suggest to create an `images` folder in project `root` then move image there and re-route.

also suggest to prefix url with `./` similar to href

codeTrackLift replied:
@nettle790

codeTrackLift commented:
Nice job w the changes, please see suggestions/comments.  Let me know w any questions, thanks!

codeTrackLift commented:
@nettle790 also friendly reminder about project tracking and keeping the project board updated.

Please review the [Project Board](https://github.com/orgs/codepharm/projects/6/views/1), you'll see that I've updated the related issue status by moving it to the In Review swim lane, and linked this as the related PR.

I've also assigned this to you as the PR `assignee`, and added this to the project board (see right sidebar of this PR).  These practices are how we stay synchronized in an asynchronous work environment. 

Hope that makes sense, let me know with any questions.  Thanks!

nettle790 commented:
Thanks.

Sent from my iPhone

> On 10-Sep-2022, at 10:43 PM, Pete Chu ***@***.***> wrote:
> 
> ﻿
> @nettle790 also friendly reminder about project tracking and keeping the project board updated.
> 
> Please review the Project Board, you'll see that I've updated the related issue status by moving it to the In Review swim lane, and linked this as the related PR.
> 
> I've also assigned this to you as the PR assignee, and added this to the project board (see right sidebar of this PR). These practices are how we stay synchronized in an asynchronous work environment.
> 
> Hope that makes sense, let me know with any questions. Thanks!
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.

nettle790 committed:
Revert "added  contents to readme.md"

This reverts commit c586f98755fda815c54e6b4264cd0385660681e6.

nettle790 committed:
added  contents to readme.md

nettle790 committed:
Merge remote-tracking branch 'origin/enhance' into enhance

nettle790 committed:
added a line of content  to README file

nettle790 committed:
Merge remote-tracking branch 'origin/enhance' into enhance

nettle790 committed:
added list item 2 in readme

nettle790 committed:
created images folder and prefixed file path

nettle790 commented on styles/styles.css:
```diff
@@ -0,0 +1,7 @@
+body {
+  background-color: pink;
+  text-align: center;
+  background-image: url("./images/athens0.jpeg");
```
url file path changed to ./

codeTrackLift replied:
Please move the `images` folder out of the `styles` folder.  See [this comment](https://github.com/codepharm/git/pull/6#pullrequestreview-1103154662)

nettle790 committed:
break tag deleted and added margin to stylesheet

codeTrackLift reviewed (commented):
Please move `images` folder out of `styles` folder, such that it is a sibling to the `styles` folder and is located at `~/images` with `~` being `root`.

nettle790 commented:
please cite one good reason as to why the images folder should be in the root directory??

codeTrackLift commented:
> please cite one good reason as to why the images folder should be in the root directory??

The images should not be under `styles` as they are not styles. To clarify, I meant the project root, not your local root.

nettle790 committed:
added root folder

nettle790 commented:
I know what a sibling folder and a tilde is meant for but I am not able to implement it practically.please review to note whether the folder structure is what it should be.

codeTrackLift commented:
```
.    
├── ...
├── styles                 
│   └── styles.css     
├── images
│   └── filename.jpg        # this can be whatever your image is called
├── .gitignore
├── index.html          
├── LICENSE     
└── README.md
```

Please remove the `root` follow the suggested folder structure above.

nettle790 committed:
filepath structure changed

codeTrackLift commented on index.html:
```diff
@@ -1,12 +1,17 @@
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+  <link rel="stylesheet" href="base/styles/styles.css" type="text/css">
```
Thanks for making the file structure updates.  I think we may need to update the `href` here as well.  Suggest to replace `base` with `./`

nettle790 committed:
deleted base folder

nettle790 commented:
please review and send me links to  any docs on relative file paths.so far I have been using the text editor right click relative file path option.this is a bit confusing.

nettle790 commented:
please review the new file structure.is it the right way to create it?
<img width="475" alt="Screenshot 2022-10-07 at 5 18 03 PM" src="https://user-images.githubusercontent.com/73188561/194547284-5bcdccf5-92e8-49be-a596-613ecf1732ce.png">

for example in the above file path could you please  reply as to which is the:

1. root directory
2. home directory 
3. project root.

codeTrackLift commented:
It looks fine, but anything beyond the project folder structure is up to you as that's on your local machine.  

For the changes we did here, I would just propose to keep file type and folder names aligned, an example is [this file structure model](https://github.com/codepharm/git/pull/6#issuecomment-1259900459).

nettle790 commented:
thanks so much.Learning filepaths now  which is important though you left one question unanswered.
Please check why index.html is conflicting in new pull request. I had deleted it and added it again in the project folder.should not be causing an issue though????

codeTrackLift commented:
It looks like a `project` directory was added and the content in `index.html` was deleted [here](https://github.com/codepharm/git/pull/11/files)

This "git practice" repo is intended for folks to practice `git` commands.

Feel free to continue building things out and take your project as far as you'd like to do, but it's probably not necessary to to merge in this 'practice' repo.  I'm going to close #11 to resolve the conflict, let me know with any questions/concerns.

nettle790 commented:
sorry ,was practicing PR's.
